.gitmodules

```diff
@@ -1,9 +1,11 @@
 [submodule "djinni"]
 	path = djinni
 	url = https://github.com/LedgerHQ/djinni.git
+# temporary fix for ios-nocodesign meanwhile they accept my PR
```

Comment:
Could you please create a fork on LedgerHQ ? In case we have other changes or/and their review takes time ?

Reply:
Sure!